tests/test_data_integrity.py

```diff
@@ -179,18 +179,33 @@ def validator(data):
 
         interfaces.library.validate(validator, success_by_default=True)
 
+    @missing_feature(library="cpp", reason="not implemented yet")
+    @missing_feature(library="dotnet", reason="not implemented yet")
+    @missing_feature(library="java", reason="not implemented yet")
+    @missing_feature(library="nodejs", reason="not implemented yet")
+    @missing_feature(library="php", reason="not implemented yet")
+    @missing_feature(library="ruby", reason="not implemented yet")
+    @missing_feature(context.library < "golang@1.73.0-dev", reason="Implemented in v1.72.0")
```

Comment:
It'll be released as part of `v1.72.0` but it was merged after branching the release branch.